app/components/admin/edit_settings_component.py

```diff
@@ -7,10 +7,45 @@ def edit_settings_component() -> str | Response:
     edit_settings_form = EditSettingsForm()
 
     if edit_settings_form.validate_on_submit():
-        return redirect(url_for("admin.settings"))
 
-    attributes = {
-        "edit_settings_form": edit_settings_form,
-    }
+        restaurant_name = edit_settings_form.restaurant_name.data
```

Comment:
would be a good idea to move this piece of code to a separate function, see how is it done in other components.